integration_tests/e2e/login.cy.ts

```diff
@@ -24,49 +20,15 @@ context('SignIn', () => {
     Page.verifyOnPage(AuthSignInPage)
   })
 
-  it('User name visible in header', () => {
```

Comment:
"Signing in" and rendering an page is no longer a responsibility of the CIAG UI, so many of these tests have been deleted.
After signing in to the CIAG UI, we now expect a redirect to the PLP UI. Given that its a redirect and the page itself (and its associated markup etc) is the responsibility of the PLP UI, we cannot (and there is no point) testing things like whether the username is displayed etc.
All of that is tested in the PLP UI 👍